app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Log in
+.row 
+  .col-md-6.col-md-offset-3
+    = form_with(url: login_path, scope: :session, local: true) do |f|
+      = f.label :email
+      = f.email_field :email, class: 'form-control'
+      = f.label :password
```

Comment:
For readability reasons, it's a good idea to group those by leaving a new line. See bellow:

```
       = f.label :email
       = f.email_field :email, class: 'form-control'

       = f.label :password
       = f.password_field :password, class: 'form-control'

       = f.submit "Log in", class: 'btn btn-primary'
```